Zend/zend_vm_def.h

```diff
@@ -1095,6 +1095,150 @@ ZEND_VM_C_LABEL(assign_op_object):
 	ZEND_VM_NEXT_OPCODE_EX(1, 2);
 }
 
+ZEND_VM_HANDLER(200, ZEND_INIT_OBJ, VAR, CONST, CACHE_SLOT, SPEC(OP_DATA=CONST|TMP|VAR|CV))
```

Comment:
This seems to only support `clone $variable with {...}`.

I assume there's more work planned to support `clone function_result() with {...}` (TMP/TMPVAR)

Reply:
Actually, I wasn't aware of this, since it's my first time when I touch the VM. Yeah, `function_result()` should work as well, so I'll need to work on this for sure.

Reply:
Ok, so I've just tested (and just pushed the change) that the clone operand can be an expression. Did you say so because of the `VAR` op1? This is really something I'll have experiment more with. originally, I used `TMPVAR`, but that didn't work (maybe because of unrelated reasons).